webapp/pages/profile/_id/_slug.vue

```diff
@@ -238,17 +238,19 @@ import HcEmpty from '~/components/Empty.vue'
 import ContentMenu from '~/components/ContentMenu'
 import HcUpload from '~/components/Upload'
 import HcAvatar from '~/components/Avatar/Avatar.vue'
-import PostMutationHelpers from '~/mixins/PostMutationHelpers'
 import PostQuery from '~/graphql/UserProfile/Post.js'
 import UserQuery from '~/graphql/UserProfile/User.js'
+
 const tabToFilterMapping = ({ tab, id }) => {
   return {
     post: { author: { id } },
     comment: { comments_some: { author: { id } } },
     shout: { shoutedBy_some: { id } },
   }[tab]
 }
+
 export default {
+  name: 'HcUserProfile',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-14T11:00:53Z" title="Friday, June 14th 2019, 1:00:53 pm +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -253,6 +252,7 @@ const tabToFilterMapping = ({ tab, id }) => {
 }
 
 export default {
+  name: 'HcUserProfile',
```


did we want to keep this change?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-14T16:43:16Z" title="Friday, June 14th 2019, 6:43:16 pm +02:00">Jun 14, 2019</time>_

---


Yes